pluma/stream/__init__.py

```diff
@@ -14,8 +14,10 @@ class StreamType(Enum):
 	MICROPHONE = 'MicrophoneStream'
 	EMPATICA = 'EmpaticaStream'
 	EEG = 'EegStream'
+	ZEROMQ = 'ZeromqStream'
```

Comment:
```suggestion
	ZEROMQ = 'ZmqStream'
```

Just marking this string so we don't forget to change it if we do the name change.